CommonLibSF/include/SFSE/API.h

```diff
@@ -19,4 +19,29 @@ namespace SFSE
 
 	Trampoline& GetTrampoline();
 	void        AllocTrampoline(std::size_t a_size, bool a_trySFSEReserve = true);
+
+	namespace stl
```

Comment:
I'm not sure whether this is the correct place to implement these helpers. This isn't really part of an API.

Reply:
Fosho where do you think they should be?

Reply:
Perhaps an `Utilities` header